.circleci/config.yml.in

```diff
@@ -834,6 +834,8 @@ jobs:
             conda install -v -y -c file://$HOME/workspace/conda-bld torchaudio
             conda install -y pandoc 'ffmpeg<5'
             apt update -qq && apt-get -qq install -y git make
+            conda install -y gxx_linux-64
```

Comment:
what is this??

Reply:
installing pesq requires `cpython` and `gxx_linux-64`, otherwise the installation will fail.

Reply:
If that's the case, can we perhaps install cpython from conda?

Reply:
there seems no cpython in conda channel. can we conda install with the github repo link?

Reply:
can you merge this installation with `conda install pandoc ffmpeg`?